triton-client/src/components/Landing/index.tsx

```diff
@@ -49,9 +50,48 @@ function LandingPage({ isLoggedIn, userName }: UserInformationProps) {
                     />
                 </Tooltip>
             </Content>
-            <Text className="Common-subtitle Landing-user" type="secondary">
-                {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
-            </Text>
+            <Content className="landing-page-content">
+                <Text className="Common-subtitle Landing-user" type="secondary">
+                    {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
+                </Text>
+                <Typography.Title level={2}>How to stage</Typography.Title>
+                <Content className="faq-container">
+                    <div className="faq-section-container">
+                        <Typography.Title level={5}>
+                            1. You will be able to stage and/or unstage the
+                            datasets on either SFTP or GLOBUS. We strongly
+                            suggest to use GLOBUS as it is faster than SFTP.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            2. You will be able to stage with either one or the
+                            other of the 2 methods available (via SFTP or
+                            GLOBUS) for each dataset. Once the datasets are
+                            staged, you will a fixed amount of time (7 days) to
+                            download the files before they get automatically
+                            unstaged from the server. It is necessary that you
+                            unstage manually (click on the same button again)
+                            the files once you have finished the download in
+                            order to continue staging and then download other
+                            files.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            3. There are 4 request status when staging the
+                            datasets. Unstaging, Stage, Queued and Failed. If
+                            you see Queued, it means that the dataset is waiting
+                            to have enough space in the disc to start staging.
+                            If you see Failed, you can communicate with{" "}
+                            <Link to={""}>{"it.genome@mail.mcgill.ca"}</Link>
+                        </Typography.Title>
```

Comment:
There might be a way to setup the link such a way that it will automatically open an email client on the user side.

Reply:
i think you are thinking of the "mailto:" URL prefix. About the citation ... you could use "hercules@mcgill.ca". It is shared by most people supporting the production (service, and us at least ...).

Reply:
i added that initially but it prompted me to use the default email service

since I use a different emailing service, i also concluded that other users will have a preferred one as well.

im not a fan of adding that but if you guys think it would be nice i can add it

Reply:
The user should be able to pick their preferred default email client, so I don't see that as an issue.